randobot/handler.py

```diff
@@ -87,14 +87,14 @@ async def begin(self):
         self.heartbeat_task = create_task(self.heartbeat(), name=f'heartbeat for {self.data.get("name")}')
         if not self.state.get('intro_sent') and not self._race_in_progress():
             await self.send_message(
-                'Welcome to OoTR! Create a seed with !seed <preset>'
+                'Welcome to OoTR! Create a release seed with !seed <preset> or a latest dev seed with !seeddev <preset>'
             )
             await self.send_message(
                 'If no preset is selected, weekly settings will be used. '
-                'Use !spoilerseed to generate a seed with a spoiler log.'
+                'Use !race to generate a release seed with a spoiler log.'
```

Comment:
Looks like `!race` hasn't actually been implemented?

Reply:
Hm I just made it consistent cause line 194 already called it !race before, but probably changed it in the wrong place lol.